src/main/resources/data.sql

```diff
@@ -1,16 +1,18 @@
 insert into term (term_id, tyear, semester, add_date, add_deadline, drop_deadline, start_date, end_date) values
-(5, 2022, 'Spring', '2021-11-01', '2022-01-30', '2022-01-30', '2022-01-15', '2022-05-17'),
+(5, 2022, 'Spring', '2021-11-01', '2022-01-30', '2022-01-30', '2022-01-15', '2022-08-14'),
 (6, 2022, 'Fall',   '2022-05-01', '2022-08-30', '2022-08-30', '2022-08-15', '2022-12-17'),
-(7, 2023, 'Spring', '2022-11-01', '2023-01-30', '2023-01-30', '2023-01-15', '2023-05-17'),
+(7, 2023, 'Spring', '2022-11-01', '2023-01-30', '2023-01-30', '2023-01-15', '2023-08-14'),
 (8, 2023, 'Fall' ,  '2023-05-01', '2023-08-30', '2023-08-30', '2023-08-15', '2023-12-17'),
-(9, 2024, 'Spring', '2023-11-01', '2024-04-30', '2024-04-30', '2024-01-15', '2024-05-17'),
+(9, 2024, 'Spring', '2023-11-01', '2024-03-30', '2024-03-30', '2024-01-15', '2024-08-14'),
 (10, 2024, 'Fall',  '2024-05-01', '2024-08-30', '2024-08-30', '2024-08-15', '2024-12-17');
```

Comment:
Another term adjustment to allow a Spring term during a summer date (for testing) without having of add a ton of data for a new term.